.github/workflows/algolia.yml

```diff
@@ -0,0 +1,28 @@
+# Updates the Algolia search index on all content changes
+
+name: Update Algolia index
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - content/**.txt
+    - site/config/algolia.php
+    - site/plugins/**
```

Comment:
Just wondering if we should add a few more folders.

````
- site/config/*
- site/models/*
- site/controllers/*
- site/templates/*
````

Or whole `site` :))
````
- site/**
````


For example, algolia indexes will not be affected when the following scenarios occur 🤔 

- a new handled data may come with the model (site/models)
- a new section can be added to the template (site/templates)
- a new route or redirection can be added from the config (site/config)

Reply:
Templates and routes don't affect the Algolia indexing. The plugin just loops over all pages in the `$site->index()` and collects the field values, some of them run through KirbyText.

I have added the models as these could in theory indeed affect the indexing result.

Reply:
@lukasbestle wouldn't template changes affect it too (e.g. if we decide to add another field)?

Reply:
@distantnative No, the fields to use are only configured in `site/config/algolia.php`.

Reply:
@lukasbestle True, thought for a moment it'd actually parse the pages, but you're right it parses just the content files based on the config.